src/main/java/seedu/address/model/wedding/ContactMap.java

```diff
@@ -14,12 +16,17 @@
  * Guarantees: details are present and not null, field values are validated.
  */
 public class ContactMap {
-    private HashMap<Role, Person> map;
+    private Map<Role, Person> map;
+    private Person husband; // Add husband
+    private Person wife;    // Add wife
```

Comment:
I don't think this is necessary, because we can access the contacts of husband and wife directly. It also separates the contacts between vendors and clients.